src/lore_session.rs

```diff
@@ -96,19 +98,19 @@ impl LoreSession {
 
         for message_id in processed_patches_ids {
             patch = self.processed_patches_map.get(&message_id).unwrap();
-            patch_number_in_series = patch.get_number_in_series();
+            patch_number_in_series = *patch.number_in_series();
 
             if patch_number_in_series > 1 {
                 continue;
             }
 
             if patch_number_in_series == 1 {
-                if let Some(in_reply_to) = &patch.get_in_reply_to() {
+                if let Some(in_reply_to) = &patch.in_reply_to() {
                     if let Some(patch_in_reply_to) =
                         self.processed_patches_map.get(&in_reply_to.href)
                     {
-                        if (patch_in_reply_to.get_number_in_series() == 0)
-                            && (patch.get_version() == patch_in_reply_to.get_version())
+                        if (*patch_in_reply_to.number_in_series() == 0)
+                            && (*patch.version() == *patch_in_reply_to.version())
```

Comment:
Including the other attributes that are of the type `usize`, I've noticed that the getters return an immutable borrow to the value, which isn't necessary as simple scalars like `usize` implement the `Copy` trait and trivially are copied to the stack. Since dereferencing is also a more "obscure" Rust feature, we should stick with the previous implementation but remove the `get_` prefixes (as @OJarrisonn pointed out, this is the overall style).

I think this is the only real adaptation I did to your change besides covering the `Config` type getters. I've done this for the `usize` attributes of `Patch` and `Config`.